### DISCUSSION TITLE
I can not fin config.yml to get wazuh dashboard node name

### DISCUSSION THREAD

**Discussion by codefaztaz** [wazuh/wazuh#19558] [Q&A]
I nedd to install a certificate but I can not figure out the Replace <dashboard-node-name> with your Wazuh dashboard node name, the same one used in config.yml to create the certificates, and move the certificates to their corresponding location.

I need to install the certificate to access the darshboard https://localhost:.....

**Comment by mdiego92** [comment]
Hello @codefaztaz,

Could you please tell me a little more about your issue? Do you need to create certificates for a new brand installation or to replace the current ones?

For example, in this part of our documentation, you would need to execute NODE_NAME=dashboard

![image](https://github.com/wazuh/wazuh/assets/62477966/a41c84cc-7d7e-4b3f-803a-f871b368a43e)

The parameter would be the same as you used in your initial config.yml file to deploy the certificates from step: https://documentation.wazuh.com/current/installation-guide/wazuh-indexer/installation-assistant.html

![image](https://github.com/wazuh/wazuh/assets/62477966/b4b61c49-f3f7-447c-b7d5-f54572a1d65e)

Looking forward to your feedback!
